README.md

```diff
@@ -258,10 +267,8 @@ A curated list of awesome Java frameworks, libraries and software.
 * [Auto](https://github.com/google/auto) - Collection of source code generators.
```

Comment:
Oh, I just noticed that `Auto` exists here too. I think it should be removed here. (The entry under Code Generators can stay.)